runtime/runc_jailer.go

```diff
@@ -200,6 +199,48 @@ func (j *runcJailer) BuildJailedMachine(cfg *config.Config, machineConfig *firec
 	return opts, nil
 }
 
+func (j *runcJailer) bindMountFIFO(src, base string) (string, error) {
+	root := j.RootPath()
+
+	if strings.ContainsRune(base, os.PathSeparator) {
+		return "", fmt.Errorf("%q must not contain %q", base, os.PathSeparator)
+	}
+
+	if _, err := os.Stat(src); os.IsNotExist(err) {
+		return "", fmt.Errorf("%s is not existed", src)
+	}
+
+	dst := filepath.Join(root, base)
+	if err := j.bindMountFileToJail(src, dst); err != nil {
+		return "", err
+	}
+
+	return base, nil
+}
+
+// BuildBindMountFifoHandler will return a new firecracker.Handler with the function
+// that will allow bind mounting of the fifos making them visible to Firecracker.
+func (j *runcJailer) BuildBindMountFifoHandler() firecracker.Handler {
+	return firecracker.Handler{
+		Name: jailerFifoHandlerName,
+		Fn: func(ctx context.Context, m *firecracker.Machine) error {
+			logFifo, err := j.bindMountFIFO(m.Cfg.LogPath, internal.FirecrackerLogFifoName)
+			m.Cfg.LogFifo = logFifo
```

Comment:
Can you check err before using logFifo? It may not change the behavior, but I'd like to make this code consistent with others.

Reply:
Good catch. Fixed it.